pkg/build/builder/docker_test.go

```diff
@@ -0,0 +1,87 @@
+package builder
+
+import (
+	"testing"
+)
+
+func TestReplaceValidCmd(t *testing.T) {
```

Comment:
Au contraire the resulting file MUST be valid Dockerfile as it's being the source of the docker build. So I insist on having such test.